### ISSUE TITLE
Authentication with sequelize / sqlite dosen't work

### ISSUE THREAD

**Issue by carweb99** [feathersjs/feathers#1031]
I got this issue when creating the feathers chat server with authentication as described on the guide. It works fine when i use postman to create a user as in the guide with NeDB. 
But if i use sequelize and sqlite doing exact the same i got this error from Postman when trying to create a user

"name": "BadRequest",
    "message": "notNull Violation: users.email cannot be null,\nnotNull Violation: users.password cannot be null",
    "code": 400,
    "className": "bad-request",
    "errors": [

when sending this 
{
  "email": "emailusername@gmail.com",
  "password": "12345678"
}

### Steps to reproduce

(First please check that this issue is not already solved as [described
here](https://github.com/feathersjs/feathers/blob/master/.github/contributing.md#report-a-bug))

- [ ] Tell us what broke. The more detailed the better.
- [ ] If you can, please create a simple example that reproduces the issue and link to a gist, jsbin, repo, etc. This makes it much easier for us to debug and issues that have a reproducable example will get higher priority.

### Expected behavior
Tell us what should happen

### Actual behavior
Tell us what happens instead

### System configuration

Tell us about the applicable parts of your setup.

**Module versions** (especially the part that's not working):

**NodeJS version**:

**Operating System**:

**Browser Version**:

**React Native Version**:

**Module Loader**:

**Comment by carweb99** [comment]
The whole response message

{
    "name": "BadRequest",
    "message": "notNull Violation: users.email cannot be null,\nnotNull Violation: users.password cannot be null",
    "code": 400,
    "className": "bad-request",
    "errors": [
        {
            "message": "users.email cannot be null",
            "type": "notNull Violation",
            "path": "email",
            "value": null,
            "origin": "CORE",
            "instance": {
                "id": null,
                "updatedAt": "2018-09-30T19:31:50.894Z",
                "createdAt": "2018-09-30T19:31:50.894Z"
            },
            "validatorKey": "is_null",
            "validatorName": null,
            "validatorArgs": []
        },
        {
            "message": "users.password cannot be null",
            "type": "notNull Violation",
            "path": "password",
            "value": null,
            "origin": "CORE",
            "instance": {
                "id": null,
                "updatedAt": "2018-09-30T19:31:50.894Z",
                "createdAt": "2018-09-30T19:31:50.894Z"
            },
            "validatorKey": "is_null",
            "validatorName": null,
            "validatorArgs": []
        }
    ]

**Comment by daffl** [comment]
The most likely reason is that you are not setting the `Content-Type` header of the Postman request to `application/json`.

**Comment by carweb99** [comment]
Ok that sounds resonable but shouldn't it be same when doing it with NeDB. I think NeDB worked

**Comment by daffl** [comment]
Are you sure NeDB worked or did it just insert an empty object instead?

**Comment by daffl** [comment]
I just confirmed that generating a new application and then authentication using Sequelize and SQLite for the users service works when sending a Postman request with the proper headers. I also confirmed that changing the username field behaves as expected. For changing the field from `email` to something else there are three things to consider

1. With Sequelize the field has to be added to the model in `src/models/users.model.js`
2. The authentication field (`authentication.local.usernameField`) in `config/default.json` has to be updated to the same name
3. If you are using Windows and the `usernameField` should be actually called `username` it has to be escaped as `\\username` (since otherwise it would be replaced with the Windows environment variable called `username`)

**Comment by carweb99** [comment]
Hi thank's for you're effort checking this! Im on Mac most of the time but using windows also. 
I have updated the models. I learned that the hard way. I changed the users.model to chatusers.model and didn't update the config file, but It's always good to explain thing then you see it more clearly and will change the architecture a little when integrating the chat functionality and only keep messages in the chat database. I want to keep chat data in separate database if it will be 100 000 of users with a lot of chat data. Authentication is done to the application database user.
Thank's a lot Best Carsten
